Update event.md -a next proposal (#42694)
dotnet/docs: docs/csharp/language-reference/keywords/event.md

---
# event (C# reference)

> An ***event*** is a member that enables notifications to be triggered. Event users can attach executable code for events by supplying ***event handlers***. The `event` keyword declares an ***event***. The event is of a delegate type. While triggering an event all supplied event handlers are invoked. Event handlers are delegate instances added to the event and executed in response to a specific event. Event users can add or remove their event handlers to/from an event.

Reviewer: I like this.  I've got a few small grammar / voice suggestions. Mostly this changes to active voice.

```suggestion
An ***event*** is a member that enables an object to trigger notifications. Event users can attach executable code for events by supplying ***event handlers***. The `event` keyword declares an ***event***. The event is of a delegate type. While an object triggers an event, the event invokes all supplied event handlers. Event handlers are delegate instances added to the event and executed when the event is raised. Event users can add or remove their event handlers on an event.
```
